src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -57,9 +61,10 @@ public WebSearchPaneViewModel(PreferencesService preferencesService, DialogServi
         this.stateManager = stateManager;
         this.preferencesService = preferencesService;
 
-        SortedSet<SearchBasedFetcher> allFetchers = WebFetchers.getSearchBasedFetchers(
+        SortedSet<SearchBasedFetcher> allFetchers = new TreeSet<>(new CompositeSearchFirstComparator());
```

Comment:
Why do you wrap them in another TreeSet? At this point you alread have a sorted set from getSearchBasedFetchers 
`        SortedSet<SearchBasedFetcher> set = new TreeSet<>(Comparator.comparing(WebFetcher::getName)); `